zpay32/fuzz_test.go

```diff
@@ -12,9 +13,44 @@ func FuzzDecode(f *testing.F) {
 	})
 }
 
+// appendChecksum returns a string containing bech followed by its bech32
+// checksum if a checksum could be calculated. Otherwise, the function returns
+// bech unchanged.
+func appendChecksum(bech string) string {
+	// This code is based on checksum calculation in zpay32/bech32.go.
```

Comment:
nit: move this comment to the function's godoc domment? Otherwise it looks a bit lost with the extra newline after it.